test/unit/codegeneration/SourceMap.js

```diff
@@ -209,7 +211,7 @@ suite('SourceMap.js', function() {
     var outFilename = 'out.js';
     var outFileContents = scriptCompiler.write(tree, outFilename);
 
-    var expected = 'alert(a);\nalert(b);\nalert(c);\n\n//# sourceMappingURL=out.map\n';
+    var expected = 'alert(a);\nalert(b);\nalert(c);\n\n//# sourceMappingURL=out.js.map\n';
```

Comment:
I gathered from some random web site that 'folks' use file.map.language-extension.